### PR TITLE
Add CODEOWNERS for Learning Path

### PR DESCRIPTION
**Pull request by tsadler1988**
Following a discussion in the Learning Path, we would like to be able to approve PRs to the website for updating Learning Path content, if the website maintainers are happy for us to do so?

There are a couple of extra things required to make this work:
- An admin to make sure @InnerSourceCommons/website-maintainers and @InnerSourceCommons/learning-path teams have access to this repo (showing up as errors in 'Files Changed' currently)
- Make sure @InnerSourceCommons/website-maintainers team is up to date
- An admin of this repo to add branch protection rules to require CODEOWNERS (see screenshot below of another project set up with this)

image

### CODE REVIEW

**Review by rrrutledge — APPROVED** [review]
Thanks for doing this, Tom!

**Comment by voborgus** [comment]
thanks a lot @tsadler1988 for this PR.

> An admin to make sure https://github.com/orgs/InnerSourceCommons/teams/website-maintainers and https://github.com/orgs/InnerSourceCommons/teams/learning-path teams have access to this repo (showing up as errors in 'Files Changed' currently)

Granted `write` permissions for this teams. Is it enough or it's required to have higher permissions?

> Make sure https://github.com/orgs/InnerSourceCommons/teams/website-maintainers team is up to date

Not able to handle this – I have no permissions to add people to this team.

> An admin of this repo to add branch protection rules to require CODEOWNERS (see screenshot below of another project set up with this)

Done

**Comment by rrrutledge** [comment]
> I have no permissions to add people to this team.

@voborgus i just gave you permissions.  Try again 🙂

**Review by clcoffey — APPROVED** [review]

**Comment by spier** [comment]
In the meantime we have added a `CODEOWNERS` file already, so we would have to figure out how to merge those two.

**Comment by rrrutledge** [comment]
> we would have to figure out how to merge those two

Merged!  Have a look.

File: .github/CODEOWNERS
Comment (spier) [inline .github/CODEOWNERS]:
```suggestion
/content/learn/learning-path @InnerSourceCommons/learning-path
```

I would opt for specifying the absolute folder location i.e. starting at root (`/`)

**Comment by spier** [comment]
Looks good overall.

The only issue right now is that @clcoffey is not part of the [website-maintainers](https://github.com/orgs/InnerSourceCommons/teams/website-maintainers/members) team yet.
@voborgus could you add her to that team? 

Once that is done, then this PR can be merged.

**Comment by tsadler1988** [comment]
> Looks good overall.
> 
> The only issue right now is that @clcoffey is not part of the [website-maintainers](https://github.com/orgs/InnerSourceCommons/teams/website-maintainers/members) team yet. @voborgus could you add her to that team?
> 
> Once that is done, then this PR can be merged.

@rrrutledge has sorted this (thanks Russ!).

**Pull request merged**
